var/spack/repos/builtin/packages/py-pynn/package.py

```diff
@@ -30,3 +31,6 @@ class PyPynn(PythonPackage):
 
     depends_on('py-neo@0.3:0.4.1',      type=('build', 'run'), when="@:0.8.3")
     depends_on('py-neo@0.5.0:',         type=('build', 'run'), when="@0.9.0:")
+    depends_on('py-lazyarray',          type=('build', 'run'), when="@0.9.0:")
```

Comment:
Missing some dependencies in the latest version:
```python
    install_requires=['numpy>=1.8.2', 'lazyarray>=0.3.2', 'neo>=0.5.2',                  
                      'quantities>=0.12.1'],        
```
Python should be build/run. Also need a build dep on setuptools.

Reply:
"fixed"... at least I traced the origin of the `install_requires` line to `pyNN@0.9.2` since there doesn't seem to have been explicitly stated dependencies before that, I'm unsure about the before part... It works in our environment, but we just link in all dependencies, so...